Upstream-Status: Inappropriate [WR Linux specific change]

Let sdparm use header files in include/scsi inside the sysroot
instead of /usr/include/scsi.

Signed-off-by: Wang Guanghui <guanghui.wang@windriver.com>

---
 src/Makefile.am |    1 -
 1 file changed, 1 deletion(-)

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -32,7 +32,6 @@
 			../lib/sg_pt_common.c
 
 if HAVE_SGUTILS
-INCLUDES = -I/usr/include/scsi
 sdparm_LDADD = @GETOPT_O_FILES@ @os_libs@ @SGUTILS_LIBS@
 sdparm_DEPENDENCIES = @GETOPT_O_FILES@
 else
